card/db/migrate/20150220141031_create_deck_routes_file.rb

```diff
@@ -0,0 +1,14 @@
+# -*- encoding : utf-8 -*-
+
+class CreateDeckRoutesFile < ActiveRecord::Migration
+  def up
+    routes_path = File.join Wagn.root, 'config', 'routes.rb'
+    if !File.exist? routes_path
+      template_path = File.join Wagn.gem_root, 'lib', 'wagn', 'generators', 'wagn', 'templates', 'config', 'routes.erb'
+      @include_jasmine_engine = false
+      decko_path = ''
+      routes = ERB.new(File.read template_path).result binding
+      File.open(routes_path, "w") { |f|  f.write routes }
+    end
+  end
+end
```

Comment:
I think I agree, but one question.  How should we configure this difference between the two cases?  I guess it is at the config/application.rb level.  For deck apps, I think we will have a small number of lines we need to add to an existing config/application.rb.  It seems like the deck app case should not require wagn/application, and they won't inherit from Wagn::Application either.  With that in mind, I think some things in wagn/application.rb need to be in a decko file, whether engine or a railtie thing.  We're close.